### PR TITLE
update doc to show how to do a multi-stage docker build

### PR DESCRIPTION
**Pull request by mkinney**

### CODE REVIEW

**Comment by tianon** [comment]
cc @sfackler (`rust` image maintainer)

File: rust/content.md [hunk 1]
```diff
@@ -30,6 +30,26 @@ $ docker build -t my-rust-app .
...
+This creates an image that has all of the rust tooling for the image, which is 1.8gb. If you just want the compiled application:
+
+```dockerfile
+FROM rust:1.39 as builder
+WORKDIR /usr/src/myapp
+COPY . .
+RUN cargo install --path .
+
+FROM debian:buster-slim
+RUN apt-get update && apt-get install -y libssl-dev
```
Comment (tianon) [inline rust/content.md]: This could be just `libssl1.1`, couldn't it?  (It probably doesn't need the headers, etc at runtime.)

Comment (mkinney) [reply rust/content.md]: Good catch. Made change.

File: rust/content.md [hunk 2]
```diff
@@ -30,6 +30,26 @@ $ docker build -t my-rust-app .
...
+COPY . .
+RUN cargo install --path .
+
+FROM debian:buster-slim
+RUN apt-get update && apt-get install -y libssl
+COPY --from=builder /usr/src/myapp /usr/local/bin/myapp
+CMD ["myapp"]
+```
+
+Note: Some shared libraries may need to be installed as shown in the installation of the libssl-dev line above.
```
Comment (sfackler) [inline rust/content.md]: Nit: it's installing libssl, not libssl-dev.

Comment (mkinney) [reply rust/content.md]: Ha. Missed that.

Thanks.

**Pull request merged**